packages/components/src/DateTimePickers/DateRange/Inputs/Inputs.scss

```diff
@@ -0,0 +1,14 @@
+$range-picker-arrow-size: 1.2rem;
```

Comment:
Please use the sass var

Reply:
It's still ko, use `$svg-sm-size`. You can find the svg size variables in the bootstrap theme > _guidelines.scss